package.json

```diff
@@ -41,7 +41,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/leandrowd/react-responsive-carousel.git"
+    "url": "https://github.com/JonathanWbn/react-responsive-carousel.git"
```

Comment:
?

Reply:
I don't know what I'm doing, trying to get this to run in my app, using this pull request. But in the node_modules there is still the old lib.

Reply:
Ah got you.

Run `npm link` from the root of this folder. Then go to your app and run `npm link <path to the carousel folder>`.

Something like: 

```
cd react-responsive-carousel
npm link
cd ../my-app
npm link ../react-responsive-carousel
```

Then go to the carousel folder again and run `npm start`.

This will get your app to run with the latest local version of the carousel and all your changes should be up to date there.

Reply:
Perfect, that works. Thank you, I appreciate it.